

array = [1, 2, 3]

array.each do |element|
  a = element
end

puts a


# a is not available outside the scope of the iteration. So it prints an error message.  

